src/components/base/Input/index.tsx

```diff
@@ -0,0 +1,22 @@
+import type { ReactJSXElement } from '@emotion/react/types/jsx-namespace';
+import styled from '@emotion/styled';
+import type { SizeType } from '../../../types';
+import type { HTMLAttributes } from 'react';
+import { INPUT_SIZE, FONT_SIZE } from '../../../utils/constants';
+export interface InputProps extends HTMLAttributes<HTMLInputElement> {
+  fontSize: SizeType;
+  inputSize: SizeType;
+}
+
+const StyledInput = styled.input<InputProps>`
+  width: ${({ inputSize }): string => INPUT_SIZE[inputSize].width};
+  height: ${({ inputSize }): string => INPUT_SIZE[inputSize].height};
+  font-size: ${({ fontSize }): string => FONT_SIZE[fontSize]};
+  border-radius: 1rem;
+`;
+
+const Input = (props: InputProps): ReactJSXElement => {
+  return <StyledInput placeholder='Enter' {...props} />;
```

Comment:
placeholder='Enter' 를 할경우,
이후에 해당 base Input을 사용할때 다른 custom한 placeholder를 넣지 못하게됩니다.
어짜피 HTML Input태그에는 placeholder가 존재하니, 별도로 기입하지 않고 ...props에서 처리할수 있도록 두는건 어떨까요?
```suggestion
  return <StyledInput {...props} />;
```

Reply:
처리...완료...! 반영했습니다!